tests/files/extern/net/http/http.gno

```diff
@@ -52,7 +53,7 @@ type Response struct {
 
 type Client struct {
 	Transport RoundTripper
-	//CheckRedirect func(req *Request, via []*Request) error
+	// CheckRedirect func(req *Request, via []*Request) error
```

Comment:
We can safely drop this, no need to have it commented out as it doesn't suggest future functionality, but is probably part of an earlier implementation